Enterprise-CMCS/cmcs-eregulations
# EREGCSC-2585 -- Update display of resources on site homepage

Resolves [EREGCSC-2585](https://jiraent.cms.gov/browse/EREGCSC-2585)

**Description**

Updates the Recent Guidance/Rules tabs on the homepage.

**This pull request changes:**

- Removes descriptive label text from Recent Guidance tab and tweaks the text in the Recent Rules tab
   - Updated text in the Recent Rules tab should be "Includes 42 CFR 400, 430-460, 483, 600; 45 CFR 95, 155-156"
- Increases number of items displayed to five for each tab
   - This is accomplished by increasing the `pageSize` argument from `3` to `5`
- For Recent Rules tab: increases the number of title lines shown from three to nine to match settings in the reader view sidebar
   - This is accomplished by updating the `itemTitleLineLimit` injection key that is provided to all descendant components
- For Recent Guidance tab: adds Subject Chips to each result item that, when clicked, takes the user to the `/subjects` page with the clicked subject selected
   - This behavior should exactly mimic the behavior on the `/search` page when you click a Subject Chip
   - This is accomplished by re-using the `SubjectChips.vue` component that is also used in the `/search` and `/subjects` page
- Adds Cypress end to end tests where appropriate

**Steps to manually verify this change:**

1. Green check marks
2. Visit [experimental deployment](https://qq6rwz29vi.execute-api.us-east-1.amazonaws.com/dev1458)
3. Ensure that Subject Chips appear under each result item in the Recent Subregulatory Guidance tab
4. When you click a Subject Chip, ensure that you are taken to the `/subjects` page with the correct subject selected on load
5. Ensure that each tab (Recent Subregulatory Guidance/Recent Rules) has five result items
   - Note that Recent Rules list items may also have nested children
6. Ensure that the title text for each Recent Rules list item can have up to nine lines of text before the `text-overflow: ellipsis` style rule is applied
   - This style rule is respected for all widths, so if you narrow your browser window size, the title label should still max out at nine lines
7. Ensure that the Recent Subregulatory Guidance tab doesn't have any label text
8. Ensure that Recent Rules tab has the following label text:
   - "Includes 42 CFR 400, 430-460, 483, 600; 45 CFR 95, 155-156"
